src/sql/workbench/browser/modelComponents/iconUtils.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IdGenerator } from 'vs/base/common/idGenerator';
+import { removeCSSRulesContainingSelector, createCSSRule, asCSSUrl } from 'vs/base/browser/dom';
+import { URI } from 'vs/base/common/uri';
+
+const ids = new IdGenerator('model-view-component-icon-');
+
+export type IUserFriendlyIcon = string | URI | { light: string | URI; dark: string | URI };
+
+export function getIconClass(iconPath: IUserFriendlyIcon, className?: string): string {
```

Comment:
>getIconClass [](start = 16, length = 12)

This is also removing the existing CSS rules for the specified class so it's not just a getter suggest updating the name to something like "clearAndSetIconClassRules"